### ISSUE TITLE
Can't find a way to clean content-repository pv

### ISSUE THREAD

**Issue by fabricepipart** [openshiftio/openshift.io#2834]
Hi!

Actually I am facing the same issue that was described [here](https://github.com/openshiftio/openshift.io/issues/891) (but closed) on my openshift.io account. By default I have a Pipeline that is set up to clean the m2repo. But I have no way to clean the content-repository.

And I can't apply the workaround mentioned in the FAQ:
* oc project [your username]-jenkins 
* oc get pods (look for the name of the "content repository" pod, for example: "content-repository-5-3cw6j")
* oc exec -ti content-repository-5-3cw6j bash
* rm -rf /var/www/html/content/repositories

since I don't have the rights to connect inside the container (I suppose the rights for those infrastructure pods have changed).

And gofabric8 CLI could not help either, I get an error message:
```
Failed to instantiate BuildConfig clean-mvn-local-repo-5abd5aa1 in namespace fabrice-pipart due to: Internal error occurred: Error getting push/pull secrets for service account fabrice-pipart/jenkins: serviceaccounts "jenkins" not found
```

**Comment by fabricepipart** [comment]
In case it helps, here are the hints that lead me to this conclusion:

In content-repository logs:
`10.128.24.1 - - [01/Apr/2018:15:52:16 +0000] "PUT /content/repositories/staging/org/trd/app/teknichrono/frontend/1.0.27/frontend-1.0.27-swarm.jar HTTP/1.1" 500 26
--
  | 01/Apr/2018:15:52:17 +0000 [ERROR 500 /content/repositories/staging/org/trd/app/teknichrono/frontend/1.0.27/frontend-1.0.27-swarm.jar] disk quota exceeded`

**Comment by fabricepipart** [comment]
Sadly not much news in 2weeks :( 

I finally found a workaround: I disabled the deployment of my maven artefacts. I actually just need the docker images to be published to be able to start the application.
```
  <build>
    <pluginManagement>
      <plugins>
        <plugin>
          <artifactId>maven-deploy-plugin</artifactId>
          <configuration>
            <skip>true</skip>
          </configuration>
        </plugin>
      </plugins>
    </pluginManagement>
  </build>
```

But note that the wildfly booster does include this maven artefacts deployment by default.

**Comment by piyush1594** [comment]
@sthaha I am not sure. What should we do about this? The other issue mentioned is closed and I never faced this.

**Comment by fabricepipart** [comment]
Hi
I would need one of the following:
- Be able to login inside the container (oc exec -ti content-repository-5-3cw6j bash). In my case I get rejected (Could not connect to the container. Do you have sufficient privileges?)
- gofabric8 CLI can be used to clean the repository (also does not work in my case)
- have a sample pipeline that I could run to empty the content-repository

Unfortunately none of those work :(
As a consequence, I just cannot deploy my maven builds locally and the content repository is just useless.

**Comment by lordofthejars** [comment]
@sthaha Since `content-repository` is going to be removed, can we close this issue?

**Comment by sthaha** [comment]
@lordofthejars yes, we can close this as won't fix as content-repository would be going away. 
@fabricepipart I am interested to know if the `content-repository` has been of any use to you?

**Comment by fabricepipart** [comment]
@sthaha No the content repository was not necessarily useful. I did not really need to publish my mvn artefacts (only the docker push is needed).
But if the content-repository is removed, then I would expect that OpenShift.io templates take this into consideration and that fabric8 pipeline libraries would not deploy anymore. Because, here I just followed the provided templates to end in that situation.
